html/semantics/forms/form-submission-0/form-submission-algorithm.html

```diff
@@ -109,6 +109,19 @@
   assert_true(event instanceof SubmitEvent);
 }, 'firing an event named submit; form.requestSubmit(null)');
 
+promise_test(async () => {
+  let form = populateForm('');
+  let iframe = form.previousSibling;
+  let event;
+  form.addEventListener('submit', e => { event = e; });
+  form.requestSubmit(null);
+  await loadPromise(iframe);
+  assert_true(event.bubbles);
+  assert_true(event.cancelable);
+  assert_equals(event.submitter, null);
+  assert_true(event instanceof SubmitEvent);
+}, 'firing an event named submit; form.requestSubmit(null)');
```

Comment:
This needs a different name from the test above, no? It seems this would result in a harness error.

Reply:
It has a different name :) I added null as parameter in the description.

Reply:
If I search for this exact string, line 110 and 123 light up.

Reply:
I think the tests are actually identical? Maybe this was already exported somehow?

Reply:
The original was added in 61c2804bf27a7abc1b8bc406627f6bf3b3344f14...

Reply:
Ah, I completely missed that somehow! It is good that this is tested, in https://bugs.webkit.org/show_bug.cgi?id=197958 we thought it was not tested. Sorry for the noise!